inference-engine/tests/functional/plugin/cpu/single_layer_tests/eltwise.cpp

```diff
@@ -0,0 +1,259 @@
+        ::testing::Combine(
+            ::testing::ValuesIn(inShapes_4D),
+            ::testing::ValuesIn(eltwiseOpTypes),
+            ::testing::ValuesIn(secondaryInputTypes),
+            ::testing::ValuesIn(opTypes),
+            ::testing::Values(InferenceEngine::Precision::FP32),
+            ::testing::Values(CommonTestUtils::DEVICE_CPU),
+            ::testing::Values(additional_config)),
+        ::testing::ValuesIn(filterCPUSpecificParams(cpuParams_4D)));
+
+INSTANTIATE_TEST_CASE_P(CompareWithRefs_4D_FP32, EltwiseLayerCPUTest, params_4D_FP32, EltwiseLayerCPUTest::getTestCaseName);
+
+std::vector<std::vector<std::vector<size_t>>> inShapes_5D = {
+        {{2, 4, 3, 4, 1}},
+        {{2, 17, 7, 5, 4}},
+        {{2, 17, 6, 5, 4}, {1, 17, 6, 1, 1}},
+        {{2, 17, 6, 5, 1}, {1, 17, 1, 1, 4}},
+};
+
+std::vector<CPUSpecificParams> cpuParams_5D = {
+        CPUSpecificParams({nCdhw16c, nCdhw16c}, {nCdhw16c}, {}, {}),
+        CPUSpecificParams({ndhwc, ndhwc}, {ndhwc}, {}, {}),
+        CPUSpecificParams({ncdhw, ncdhw}, {ncdhw}, {}, {})
+};
+
+const auto params_5D_FP32 = ::testing::Combine(
+        ::testing::Combine(
+            ::testing::ValuesIn(inShapes_5D),
+            ::testing::ValuesIn(eltwiseOpTypes),
+            ::testing::ValuesIn(secondaryInputTypes),
+            ::testing::ValuesIn(opTypes),
+            ::testing::Values(InferenceEngine::Precision::FP32),
+            ::testing::Values(CommonTestUtils::DEVICE_CPU),
+            ::testing::Values(additional_config)),
+        ::testing::ValuesIn(filterCPUSpecificParams(cpuParams_5D)));
+
+INSTANTIATE_TEST_CASE_P(CompareWithRefs_5D_FP32, EltwiseLayerCPUTest, params_5D_FP32, EltwiseLayerCPUTest::getTestCaseName);
+
+std::vector<std::vector<std::vector<size_t>>> inShapes_4D_Mixed = {
+        {{2, 17, 31, 3}, {2, 1, 31, 3}},
+        {{2, 17, 5, 1}, {2, 1, 1, 4}},
+};
+
+std::vector<CPUSpecificParams> cpuParams_4D_Mixed = {
+        CPUSpecificParams({nChw16c, nchw}, {nChw16c}, {}, {}),
+};
+
+const auto params_4D_FP32_Mixed = ::testing::Combine(
+        ::testing::Combine(
+            ::testing::ValuesIn(inShapes_4D_Mixed),
+            ::testing::ValuesIn(eltwiseOpTypes),
+            ::testing::Values(ngraph::helpers::InputLayerType::CONSTANT),
+            ::testing::ValuesIn(opTypes),
+            ::testing::Values(InferenceEngine::Precision::FP32),
+            ::testing::Values(CommonTestUtils::DEVICE_CPU),
+            ::testing::Values(additional_config)),
+        ::testing::ValuesIn(filterCPUSpecificParams(cpuParams_4D_Mixed)));
+
+INSTANTIATE_TEST_CASE_P(CompareWithRefs_4D_FP32_Mixed, EltwiseLayerCPUTest, params_4D_FP32_Mixed, EltwiseLayerCPUTest::getTestCaseName);
+
+std::vector<std::vector<std::vector<size_t>>> inShapes_5D_Mixed = {
+        {{2, 17, 31, 4, 3}, {2, 1, 31, 1, 3}},
+        {{2, 17, 5, 3, 1}, {2, 1, 1, 3, 4}},
+};
+
+std::vector<CPUSpecificParams> cpuParams_5D_Mixed = {
+        CPUSpecificParams({nCdhw16c, ncdhw}, {nCdhw16c}, {}, {}),
```

Comment:
Can you add more cases? See comment above

Reply:
Done